src/v/cluster/members_manager.cc

```diff
@@ -1527,9 +1527,19 @@ members_manager::handle_configuration_update_request(
           req.target_node);
         return ss::make_ready_future<ret_t>(configuration_update_reply{false});
     }
+
+    auto& all_brokers = _members_table.local().nodes();
+    if (!all_brokers.contains(req.node.id())) {
```

Comment:
Nit: so the crash happens on the assertion in `check_result_configuration`, right? Maybe it should just return an error in this case instead of asserting. This way we will avoid the double check.

Reply:
good point, i will change this